server/graphql/v2/input/CollectiveCreateInput.js

```diff
@@ -6,6 +6,8 @@ export const CollectiveCreateInput = new GraphQLInputObjectType({
   fields: () => ({
     name: { type: new GraphQLNonNull(GraphQLString) },
     slug: { type: new GraphQLNonNull(GraphQLString) },
+    website: { type: GraphQLString },
+    type: { type: GraphQLString },
```

Comment:
There should not be a `type` here, if it's a `CollectiveCreateInput` then the type is `Collective`.
If we want to be able to create an organization we need to define a new `OrganizationCreateInput`.

Reply:
@Betree just to make sure I am doing the right thing, You mean I had to create a mutation in V2 for Organization, and also the file `OrganizationCreateInput`

Reply:
Does it mean I also create new mutations in the frontend.  `CreateOrganizationMutation` and `CreateOrganizationMutation`

Reply:
Yes, you'll need to create new mutations dedicated to organizations in the frontend